java-shared-dependencies/third-party-dependencies/pom.xml

```diff
@@ -139,6 +139,11 @@
         <artifactId>opentelemetry-grpc-1.6</artifactId>
         <version>${opentelemetry-grpc-instrumentation.version}</version>
       </dependency>
+      <dependency>
+        <groupId>io.opentelemetry.semconv</groupId>
+        <artifactId>opentelemetry-semconv</artifactId>
+        <version>${opentelemetry-semconv.version}</version>
+      </dependency>
       <dependency>
```

Comment:
Should the `io.opentelemetry:opentelemetry-semconv` deps below (L147-L150) be removed since it's now `io.opentelemetry.semconv:opentelemetry-semconv`?

Reply:
I had tried that originally, but saw some checks fail because the Pub/Sub library currently uses the old `io.opentelemetry:opentelemetry-semconv` dependency. I was planning to remove that old one in a follow-up PR after the dependency is removed in Pub/Sub.

Reply:
Ah, makes sense. I believe our downstream tests use the latest released version of pubsub. In that case, would you mind either creating an issue in this repo or adding a comment in the ticket so we don't forget to remove the old deps?